apps/mobile/src/state/marketplace/index.ts

```diff
@@ -540,6 +541,25 @@ export const deleteOffersActionAtom = atom<
               !adminIdsToDelete.includes(o.ownershipInfo?.adminId)
           )
         )
+
+        // Only when deleting one offer
+        if (adminIdsToDelete.length === 1) {
```

Comment:
This will run only when deleting offer. It might be good idea to iterate through all inboxes periodically and check wether they should be removed or not (if there are none offers nor chats)